.github/workflows/ci_push.yml

```diff
@@ -0,0 +1,21 @@
+name: Main workflow 
+
+on:
+ push:
+    # Sequence of patterns matched against refs/heads
+    branches:
+      - master         # Push events on master branch
+      - 'releases/*'   # Push events to branches matching refs/heads/releases/*
```

Comment:
We don't really do release branches on PINOperation, just tags. Maybe we should start though…